interface/forms/newpatient/templates/report.html.twig

```diff
@@ -0,0 +1,13 @@
+<div class="w-100">
+{% for e in encounters %}
+    {% set _reason = (e.reason) ? " - " ~ e.reason|nl2br|xl : "" %}
+    <h3>{{ e.category|xlt }}{{ _reason }}</h3>
```

Comment:
```
<h3>{{ e.category|text }}{{ _reason }}</h3>
```

Reply:
Looks like this has not been addressed yet. Rec keeping it what it was in above code where it escapes and does not translate. If want to translate it, then would need to use the xl_appt_category() function which uses the global setting `$GLOBALS['translate_appt_categories']` to decide whether to translate (allows users to decide if translate these or if they simply do their own translations directly in the label). In that case would make my recommended change above (use text). and then wrap the label in report.php line 43 with the xl_appt_category() call

Reply:
Got it, I'll use `xl_appt_category()` in the controller to make it translatable.

Reply:
hi @robertdown , this was not addressed in this PR??

Reply:
It should have been? I called that function from the controller. I'll look at the code, but I made that change before committing. If it's not there I'll add it.